### PR TITLE
Pulumi release notes 103

### PR DESCRIPTION
**Pull request by arunkumar611**

### CODE REVIEW

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Pulumi Release Notes: Pulumi ESC, Pulumi Dev Releases, and more!"
+allow_long_title: true
+date: 2024-04-12T11:06:04-08:00
+draft: false
+meta_desc: The latest Pulumi updates include Pulumi ESC, Pulumi AI, Pulumi CLI & SDK Dev Releases, and new Pulumi providers
```
Comment (meagancojocar) [inline themes/default/content/blog/pulumi-release-notes-103/index.md:6]:
```suggestion
meta_desc: The latest Pulumi updates include a new Pulumi ESC Key Value Editor, Pulumi CLI & SDK Dev Releases, and more!
```

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+title: "Pulumi Release Notes: Pulumi ESC, Pulumi Dev Releases, and more!"
```
[the remark this answers is not in the document]
Comment (meagancojocar) [reply themes/default/content/blog/pulumi-release-notes-103/index.md:2]: if this is too long you can add allow_long_title: True

**Review by meagancojocar — COMMENTED** [review]
Only got 1/3 of the way through but pushing in case I don't get to it later today!

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+### Pulumi ESC Audit Logs
+
+Pulumi ESC now features comprehensive Audit Logs for all Create, Read, Open, Update, and Delete (CRUD) operations, strengthening your cloud security framework. This critical addition to Pulumi ESC enables organizations to closely monitor and audit their cloud infrastructure changes, ensuring enhanced oversight and security compliance. Through detailed logging, teams gain valuable insights into their operational activities, facilitating improved governance and accountability across cloud resources.
+
+## Core
+
+### Dev Releases for the Pulumi CLI and SDKs
```
Comment (justinvp) [inline themes/default/content/blog/pulumi-release-notes-103/index.md:72]: While we're summarizing other blog posts, can we also add another one for Node.js Monorepos? Basically summarizing and linking to https://www.pulumi.com/blog/nx-monorepo/

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+### Automation API support to install Pulumi CLI 
+
+The Pulumi Automation API now offers the functionality to [install the Pulumi CLI](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3/go/auto#InstallPulumiCommand) directly through the API. This feature enables deploying programs utilizing the Automation API without the need for prior CLI installation. Explore the [example code](https://github.com/pulumi/automation-api-examples/tree/main/go/cli_installation) to see how to integrate the new API seamlessly.
```
Comment (justinvp) [inline themes/default/content/blog/pulumi-release-notes-103/index.md:78]: It's unfortunate we only have an example in Go. This is usable from the other languages that support automation API. Perhaps worth at least linking to the API docs for some of the other languages: 
- Node.js: https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/pulumi/classes/automation.PulumiCommand.html#install
- Python: https://www.pulumi.com/docs/reference/pkg/python/pulumi/#pulumi.automation.PulumiCommand.install
- .NET: Unfortunately, we need to manually kick off a doc update for the new API to show up in https://www.pulumi.com/docs/reference/pkg/dotnet/Pulumi.Automation/Pulumi.Automation.html. In the meantime, we could link directly to https://github.com/pulumi/pulumi-dotnet/blob/edf4a5e904110f198d579d8db0b178aa9af9cdfa/sdk/Pulumi.Automation/Commands/LocalPulumiCommand.cs#L107

File: themes/default/content/blog/pulumi-release-notes-103/index.md (hunk 3 above)
Comment (arunkumar611) [reply themes/default/content/blog/pulumi-release-notes-103/index.md:72]: Usually, we add only 'new' releases to this blog. Did we do anything to support node monorepos, or it is more of a 'how-to' guide?

Comment (justinvp) [reply themes/default/content/blog/pulumi-release-notes-103/index.md:72]: > Did we do anything to support node monorepos

Yes, we made several product enhancements to enable this (e.g. https://github.com/pulumi/pulumi/issues/12688, https://github.com/pulumi/pulumi/issues/8865)

Comment (arunkumar611) [reply themes/default/content/blog/pulumi-release-notes-103/index.md:72]: Is this summary good? Anything you'd add? 

Title: Node.js Monorepo Support 
Summary: Pulumi now seamlessly integrates with Node.js monorepos using tools like Nx. We've enhanced Pulumi to support yarn/npm workspaces, widely used in Node.js monorepos. This enhancement supports npm and Yarn workspaces within monorepos, facilitating better project management by automatically handling dependencies and build processes. This integration allows Pulumi infrastructure codes to coexist with application codes in the same repository, improving development workflow efficiency by using Nx's capabilities to manage build and deploy dependencies effectively.  Check out the [blog post](/blog/nx-monorepo/) for more details.

File: themes/default/content/blog/pulumi-release-notes-103/index.md (hunk 4 above)
Comment (arunkumar611) [reply themes/default/content/blog/pulumi-release-notes-103/index.md]: I added them

File: themes/default/content/blog/pulumi-release-notes-103/index.md (hunk 3 above)
Comment (justinvp) [reply themes/default/content/blog/pulumi-release-notes-103/index.md:72]: This looks good to me

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+### Automation API support to install Pulumi CLI 
+
+The Pulumi Automation API now offers the functionality to [install the Pulumi CLI](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3/go/auto#InstallPulumiCommand) directly through the API. This feature enables deploying programs utilizing the Automation API without the need for prior CLI installation. Explore the example code in [go](https://github.com/pulumi/automation-api-examples/tree/main/go/cli_installation), [node.js](docs/reference/pkg/nodejs/pulumi/pulumi/classes/automation.PulumiCommand.html#install), [python](/docs/reference/pkg/python/pulumi/#pulumi.automation.PulumiCommand.install), and [.NET](https://github.com/pulumi/pulumi-dotnet/blob/edf4a5e904110f198d579d8db0b178aa9af9cdfa/sdk/Pulumi.Automation/Commands/LocalPulumiCommand.cs#L107) to see how to integrate the new API seamlessly.
```
Comment (justinvp) [inline themes/default/content/blog/pulumi-release-notes-103/index.md:78]: The docs links are listed among the Go example. How about re-arranging so we call out just the Go example, and then link to API docs for all the languages? Something like

```suggestion
The Pulumi Automation API now offers the functionality to install the Pulumi CLI directly through the API. This feature enables deploying programs utilizing the Automation API without the need for prior CLI installation. Explore an [example in Go](https://github.com/pulumi/automation-api-examples/tree/main/go/cli_installation) and refer to the new APIs in [Go](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3/go/auto#InstallPulumiCommand), [Node.js](docs/reference/pkg/nodejs/pulumi/pulumi/classes/automation.PulumiCommand.html#install), [Python](/docs/reference/pkg/python/pulumi/#pulumi.automation.PulumiCommand.install), and [.NET](https://github.com/pulumi/pulumi-dotnet/blob/edf4a5e904110f198d579d8db0b178aa9af9cdfa/sdk/Pulumi.Automation/Commands/LocalPulumiCommand.cs#L107).
```

**Review by justinvp — COMMENTED** [review]
The parts for Core LGTM. Thank you!

**Review by meagancojocar — APPROVED** [review]

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
+---
+title: "Pulumi Release Notes: Pulumi ESC Table Editor, 1Password Support, Pulumi Dev Releases, and more!"
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-103/index.md:2]:
```suggestion
title: "Pulumi Release Notes: Pulumi Dev Releases, Pulumi ESC Table Editor, 1Password Support, and more!"
```

File: themes/default/content/blog/pulumi-release-notes-103/index.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+  - [Node.js Monorepo Support](#nodejs-monorepo-support)
+- [Providers and Packages](#providers-and-packages)
+  - [New Provider Resources](#new-provider-resources)
+  - [New Community Providers](#new-community-providers)
+  - [Pulumi Registry Documentation Update](#pulumi-registry-documentation-update)
+- [Wrap up](#wrap-up)
+
+## AI
+
+### Pulumi AI Evolution
```
Comment (lukehoban) [inline themes/default/content/blog/pulumi-release-notes-103/index.md:39]: Might move this to lower down the list.

**Review by lukehoban — APPROVED** [review]

**Pull request merged**